### DISCUSSION TITLE
[QuickNav] Custom link components

### DISCUSSION THREAD

**Discussion by marcialfps** [dxc-technology/halstack-react#1511] [Ideas]
Since the current QuickNav is not supported by React Router library, we need to look for a solution for the different routing libraries. A solution can be achieved by adding `component` prop like the one we have in Link component. This component will substitute the default anchor and provide more customization by the user. 

E.g. we can add HashLinks from [React Router Hash Links](https://github.com/rafgraph/react-router-hash-link) to solve integration with React Router.

**Comment by kukoc** [comment]
Hi @marcialfps . Your request is fair enough. We need to allocate some time to investigate the best approach. If you have already coded something that works for you, please share it with the team. Thanks
